### PR TITLE
New Analytics Adapter: Implement Appier Analytics Adapter.

### PR DESCRIPTION
**Pull request by kd-appier**
## Type of change

- [ ] Bugfix
- [ ] Feature
- [ ] New bidder adapter   
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Does this change affect user-facing APIs or examples documented on http://prebid.org?
- [X] Other
    *  Analytics Adapter

## Description of change
 New Appier Analytics Adapter to gather Prebid.js event data and forward to Appier reporting platform.

* apn-dev@appier.com
- [X] official adapter submission

For any changes that affect user-facing APIs or example code documented on http://prebid.org, please provide:

A link to a PR on the docs repo at https://github.com/prebid/prebid.github.io/

### CODE REVIEW

File: modules/appierAnalyticsAdapter.js [hunk 1]
```diff
@@ -0,0 +1,245 @@
+import {ajax} from 'src/ajax';
```
[the remark this answers is not in the document]
Comment (kd-appier) [reply modules/appierAnalyticsAdapter.js]: Fixed, thank you~

File: modules/appierAnalyticsAdapter.js [hunk 2]
```diff
@@ -0,0 +1,245 @@
...
+import adapter from '../src/AnalyticsAdapter';
+import CONSTANTS from '../src/constants.json';
+import adapterManager from '../src/adapterManager';
+import {logError, logInfo} from '../src/utils';
+
+const utils = require('../src/utils');
```
[the remark this answers is not in the document]
Comment (kd-appier) [reply modules/appierAnalyticsAdapter.js]: Fixed, thank you~

File: modules/appierAnalyticsAdapter.js [hunk 3]
```diff
@@ -0,0 +1,245 @@
...
+  NO_BID: 'noBid',
+  BID_WON: 'bidWon',
+  TIMEOUT: 'timeout'
+};
+
+export const getCpmInUsd = function (bid) {
+  if (bid.currency === 'USD') {
+    return bid.cpm;
+  } else {
+    return bid.getCpmInNewCurrency('USD');
```
[the remark this answers is not in the document]
Comment (kd-appier) [reply modules/appierAnalyticsAdapter.js]: Yes, currency module is required. I have added this reminder in README.md, thank you :)

**Comment by kd-appier** [comment]
Thank you @jsnellbaker :)

I have fixed the issues in your comments. This is the testing config:

```
            pbjs.enableAnalytics([
                {
                    provider: 'appierAnalytics',
                    options: {
                        affiliateId: 'Wh9T9301',
                        configId: '1293c411edca44579dfc59a1b7ac2808',
                        server: 'https://prebid-analytics-stg.c.appier.net/v1'
                    }
                }
            ]);
```

This is our PR for the docs repo: 
https://github.com/prebid/prebid.github.io/pull/1347

Thank you very much :)

**Comment by jsnellbaker** [comment]
@kd-appier Thanks for all the updates and the test credentials.  Everything appeared to be working successfully on my end and the code changes look good.

**Comment by kd-appier** [comment]
Cool. thank you @jsnellbaker :)

**Pull request merged**